tools/src/main/resources/javascript/apps/security-filter-jwt.js

```diff
@@ -94089,7 +94089,7 @@ var store = global[SHARED] || (global[SHARED] = {});
 })('versions', []).push({
   version: core.version,
   mode: __webpack_require__(/*! ./_library */ "./node_modules/core-js/library/modules/_library.js") ? 'pure' : 'global',
-  copyright: '© 2019 Denis Pushkarev (zloirock.ru)'
+  copyright: '© 2020 Denis Pushkarev (zloirock.ru)'
```

Comment:
this is an external library, so I don't know if we should update the copyright like that

Reply:
Right, good catch